get-started.md

```diff
@@ -1,5 +1,69 @@
 ---
 layout: get-started
-title: Get Started
+title: Kogito - Get Started
 permalink: /get-started/
 ---
+
+<div class="grid__item width-4-12 hide-mobile toc" markdown="1">
+## Table of Contents
+### Getting Started
+
+ - [Kogito business automation overview](https://docs.jboss.org/kogito/release/latest/html_single/#con_kogito-automation)
```

Comment:
shouldn't this be linked with the ## Kogito business automation overview topic as this topic already points to the same doc?

Reply:
No this is how it should behave as we agreed on with @sterobin  ... directly linking to the docs.